src/pudl/package_data/censuspep/territories.csv

```diff
@@ -0,0 +1,21 @@
+state_id_fips,county_id_fips,area_name,fips_level,report_date
```

Comment:
Nonblocking: Could you commit the/a script for generating this file? That way it's easier to update if we need to (perhaps in 6 years; the ANSI files had updates for the last two census-es so it seems likely they'll have one for 2030)

Reply:
good idea! I did this pretty manually bc there were so few of em. but i added details about that process into the transform function doc string